api/handlers/exporter.go

```diff
@@ -0,0 +1,102 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	spectypes "github.com/ssvlabs/ssv-spec/types"
+
+	"github.com/ssvlabs/ssv/api"
+	exporterapi "github.com/ssvlabs/ssv/exporter/api"
+	"github.com/ssvlabs/ssv/exporter/convert"
+	ibftstorage "github.com/ssvlabs/ssv/ibft/storage"
+)
+
+type Exporter struct {
+	DomainType spectypes.DomainType
+	QBFTStores *ibftstorage.QBFTStores
+}
+
+type ParticipantResponse struct {
+	Role      string `json:"role"`
+	Slot      uint64 `json:"slot"`
+	PublicKey string `json:"public_key"`
+	Message   struct {
+		Signers []uint64 `json:"Signers"`
+	} `json:"message"`
+}
+
+func (e *Exporter) Decideds(w http.ResponseWriter, r *http.Request) error {
+	var request struct {
+		From    int           `json:"from"`
+		To      int           `json:"to"`
+		Roles   api.RoleSlice `json:"roles"`
+		PubKeys api.HexSlice  `json:"pubkeys"`
+	}
+	var response struct {
+		Data []*ParticipantResponse `json:"data"`
+	}
+
+	if err := api.Bind(r, &request); err != nil {
+		return err
+	}
+
+	if request.From > request.To {
+		return api.BadRequestError(fmt.Errorf("from must be less than to"))
+	}
+	if request.From < 0 || request.To < 0 {
+		return api.BadRequestError(fmt.Errorf("from and to must be greater than 0"))
+	}
+	if len(request.PubKeys) == 0 {
+		return api.BadRequestError(fmt.Errorf("at least one public key is required"))
+	}
+	if len(request.Roles) == 0 {
+		return api.BadRequestError(fmt.Errorf("at least one role is required"))
+	}
+
+	response.Data = []*ParticipantResponse{}
+	for _, role := range request.Roles {
+		roleStorage := e.QBFTStores.Get(convert.RunnerRole(role))
```

Comment:
It's not introduced in this PR but `convert.RunnerRole` has 
```go
	RoleSyncCommitteeContribution = 3
	RoleSyncCommittee = 4
```

but `spectypes.BeaconRole` has 

```go
	BNRoleSyncCommittee = 3
	BNRoleSyncCommitteeContribution = 4
```

, so those roles seem to be inverted here. However, it isn't not part of the code of this PR

Reply:
@MatusKysel could you please confirm that it's not intended?